pep-0458.txt

```diff
@@ -1044,7 +1044,7 @@ References
 ==========
 
 .. [1] https://pypi.python.org
-.. [2] https://isis.poly.edu/~jcappos/papers/samuel_tuf_ccs_2010.pdf
+.. [2] https://theupdateframework.github.io/papers/survivable-key-compromise-ccs2010.pdf
 .. [3] http://www.pip-installer.org
 .. [4] https://wiki.python.org/moin/WikiAttack2013
 .. [5] https://github.com/theupdateframework/pip/wiki/Attacks-on-software-repositories
```

Comment:
FYI, I left this link. although the wiki page it points to doesn't list any attacks after 2016. I did, however, update that wiki page to direct the reader to a broader and more up to date [collection of supply chain compromises](https://github.com/in-toto/supply-chain-compromises).